### PR TITLE
Feat/Homepage

### PR DESCRIPTION
**Pull request by soerenstein**
This pull request is linked to #2 
Please review the following files of my user story:

HomePage.js
HomePage.stories.js
NavButton.js
NavButton.test.js
WelcomeUser.js
TopLeftCard.js
CardWrapper.js

Thank you!

### CODE REVIEW

File: src/components/Design/CardWrapper/CardWrapper.js [hunk 1]
```diff
@@ -6,10 +6,10 @@ export default function CardWrapper() {
 }
 
 const StyledCardWrapper = styled.div`
-  position: absolute;
+    position: absolute;
```
Comment (ninasander) [inline src/components/Design/CardWrapper/CardWrapper.js:9]: I think you accidentally added a space here ;)

File: src/components/Design/CardWrapper/TopLeftCard.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+
+export default function TopLeftCard() {
+    return (
+<>
+        <StyledTopCard /> 
+</>
```
Comment (ninasander) [inline src/components/Design/CardWrapper/TopLeftCard.js:8]: You don't need the fractions around it when you only return one element.

File: src/components/HomePage/HomePage.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import WelcomeUser from './WelcomeUser'
+import TopLeftCard from '../Design/CardWrapper/TopLeftCard'
+import NavButton from './NavButton'
+
+export default function HomePage({ user }) {
+  return (
+    <div>
```
Comment (ninasander) [inline src/components/HomePage/HomePage.js:9]: I think a fraction instead of a div would be enough unless you still want to style it later.

**Review by ninasander — COMMENTED** [review]
Nice work! Just added some minor suggestions :)

**Review by jennydely — APPROVED** [review]
looks good

**Comment by soerenstein** [comment]
> Nice work! Just added some minor suggestions :)

Thanks for your suggestions! :)

File: src/components/HomePage/HomePage.js (hunk 3 above)
Comment (soerenstein) [reply src/components/HomePage/HomePage.js:9]: Thanks for your suggestions! :)

**Pull request merged**